lib/routes.jsx

```diff
@@ -115,6 +115,8 @@ var routeElements = Object.keys(pages).map(function(path) {
   return <Route path={path} component={pages[path]} key={path}/>;
 });
 
+var makerPartyLocales = ['de', 'fr', 'es', 'pl', 'it', 'nl', 'cs', 'bg'];
```

Comment:
I guess it depends on how learning.m.o works when it comes to fr-FR and fr.

We want France French, and Spain Spanish, so whatever codes fall under that.

Interestingly enough, in donate, fr-FR isn't a thing, and es-ES isn't even complete, so maybe it's low priority.